backend/src/main/resources/data.sql

```diff
@@ -1,47 +1,26 @@
--- CLEAR
-
-UPDATE SURVEY -- REMOVE IT BECAUSE WE WILL RUN INTO CONSTRAINT PROBLEMS OTHERWISE
-SET CHOSEN_TIMESLOT_ID=NULL
-WHERE ID = 6;
-
-DELETE
-FROM RESPONSE_TIMESLOT
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM RESPONSE
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM TIMESLOT
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM PARTICIPATION
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM SURVEY
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM USER
-WHERE ID <= 26
-  AND ID >= -1;
-
-
--- UNUSED USERS
-
--- PW = password4
-INSERT INTO USER (ID, EMAIL, FULL_NAME, PASSWORD_HASH)
-VALUES (-1, 'Name-1@examle.com', 'Name-1', '$2a$10$upLmX5blX639STzhEuLwZ.PiFLHEBJXeglUPuKiYLpxvTvyyI1ppm');
+---------------------
+-- ID Ranges
+-- User:                00 - 09
+-- Survey:              10 - 19
+-- Participations:
+--        from Creator: 20 - 29
+--       with Response: 30 - 39
+--    without Response: 40 - 49
+-- Responses:
+--        from Creator: 50 - 59
+--               other: 60 - 69
+-- Timeslots:           70 - 99
+-- ResponseTimeslots:
+--        from Creator: 100 - 139
+--               other: 140 - ? (make sure to update HIBERNATE_SEQUENCE START)
```

Comment:
What's the rationale behing those newly introduced changes? Simply buffering for future changes?

Reply:
Yes. Just to have some more space and to make it easier to directly infer the type of the object by looking at the id.